databases/week1/db-homework1.sql

```diff
@@ -0,0 +1,29 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT(*) AS totalTasks FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT title, due_date FROM task  WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * FROM task WHERE status_id = (SELECT id FROM status WHERE name = 'done');
+
+-- Find all the tasks that are not marked as done
+SELECT * FROM task WHERE status_id IN (SELECT id FROM status WHERE name != 'done');
+
```

Comment:
Here I want to see states' names. How do I know what status id's 1,2 or 3 are?  Try something like this and see if it reflects the request.
```
SELECT * FROM task
JOIN status ON task.status_id = status.id 
WHERE status.name = 'Done';
```

Reply:
Thank you for the feedback I will make the suggested changes.